remindme-notifier/src/main/java/it/andreafailli/remindme/notifier/controllers/CronHandlerController.java

```diff
@@ -24,11 +27,21 @@
 	public ResponseEntity<Void> handler() {
 		LOGGER.entry();
 		
-		//TODO: list current reminders and send notifications
+		LocalDateTime now = LocalDateTime.now();
+		
+		Iterable<Reminder> reminders = this.reminderService.list(now);
+		for (Reminder reminder : reminders) {
+			this.sendNotification(reminder);
+		}
+		
 		ResponseEntity<Void> response = ResponseEntity.ok().build();
 		
 		LOGGER.exit(response);
 		return response;
 	}
+	
+	private void sendNotification(Reminder reminder) {
+		// TODO: send notification
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)